api/src/main/scala/hmda/api/http/ValidationErrorConverter.scala

```diff
@@ -53,7 +54,17 @@ trait ValidationErrorConverter {
       (fieldName, toJsonVal(fieldValue))
     }
 
-    JsObject(jsVals: _*)
+    // Add MSA/MD name to edits
+    val jsValsWithMsaName = if (err.ruleName == "Q595" || err.ruleName == "Q029") {
```

Comment:
What if we added a field to the "relevant fields" column of the `edit-metadata` file for `Q595` and `Q029`, (something like "Metropolitan Statistical Area / Metropolitan Division Name")? I bet, in that case, we could make the `LarFieldMapping` module responsible for finding the MSA/MD name, we wouldn't have to have a special case for these two edits here in the `ValidationErrorConverter`